data-extraction/pyDataExtraction/__main__.py

```diff
@@ -0,0 +1,10 @@
+from json import dumps
+from typing import Union, Dict, Optional
+from abc import ABC, abstractmethod
+from pyDataExtraction.commonTypes.Graph import Graph
+
+if __name__ == "__main__":
+    graph_data1 = {"A": ["B", "C"], "B": ["A,C"], "C": ["A,D"], "D": ["A"]}
+    graph_data2 = {1: [2, 3], 2: [1, 3], 3: [1, 4], 4: [1]}
+    graph = Graph(graph_data1)
+    print(graph)
```

Comment:
Please use logging with respective log level

Reply:
I'm not sure what you mean. I was mainly using `__main__` to verify that the graph implementation works as I was writing it, though that should be move to a separate directory for testing, as seems to be the standard practice for python libraries. 

I'm not sure if we could use the same test directory as node without pytest throwing errors due to the presence of js files. my experience with pytest and how it works is limited.